src/main/java/dev/codesupport/web/api/controller/UserController.java

```diff
@@ -2,38 +2,33 @@
 
 import dev.codesupport.web.common.service.service.RestResponse;
 import dev.codesupport.web.domain.User;
+import dev.codesupport.web.domain.UserRegistration;
+import dev.codesupport.web.domain.UserStripped;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.List;
+import javax.validation.Valid;
 
 /**
  * Defines endpoints and validations for the associated API Contract for the {@link User} resource.
  */
 @RestController
-@RequestMapping("/api/v1")
+@RequestMapping("api/user/v1")
```

Comment:
Surely it would be a `/api/v1/user`?

Reply:
actually it would be
``api/user/v1/users`` though I'm not sure the api part is really needed.  and the other should be
``api/user/v1/profiles`` since they are both resources that are part of the user